src/presentation/ui/component/form/inputField.templ

```diff
@@ -67,22 +67,23 @@ templ InputFieldWithPrefix(
 	</div>
 }
 
+// Note: if "clientSideValuePath" is provided, "serverSideValue" will be ignored.
 templ InputFieldReadOnly(
-	inputType, id, label, bindValuePath string,
-	denseMode bool,
+	inputType, id, label, clientSideValuePath, serverSideValue string,
 ) {
 	<!-- InputFieldReadOnly -->
 	<div class="relative w-full">
 		<input
 			type={ inputType }
 			id={ id }
 			name={ id }
-			x-model:value={ bindValuePath }
-			if denseMode {
-				class="bg-os-300 border-os-200 hover:border-os-100 autofill:bg-os-300 focus:border-os-50 h-7.5 peer relative w-full cursor-default rounded-md border px-1.5 text-[13px] text-slate-400 placeholder-transparent outline-none transition-all"
-			} else {
-				class="bg-os-300 border-os-200 hover:border-os-100 autofill:bg-os-300 focus:border-os-50 peer relative h-10 w-full cursor-default rounded-md border px-3 text-sm text-slate-400 placeholder-transparent outline-none transition-all"
+			if clientSideValuePath != "" {
+				x-model:value={ clientSideValuePath }
+			}
+			if serverSideValue != "" {
+				value={ serverSideValue }
 			}
```

Comment:
For this, I'd like to discuss with you about this. I'm definitely struggling to find a good, easily readable solution for this situation. We had discussed doing something similar to the searchable component you created, but in this case, it doesn't work because I'm dealing with `[]string` values. Even if I add it, I would always need to use the searchable struct and never use `[]string` again. It feels like I'm using a bazooka to kill a fly, you know?